paasta_tools/check_marathon_services_replication.py

```diff
@@ -26,7 +26,7 @@
 
 After retrieving that information, a fraction of available instances is calculated
 (available/expected), and then compared against a threshold. The default threshold
-is .50, meaning if fewer than 50 of a service's backends are available, the script
+is 50, meaning if less than 50% of a service's backends are available, the script
```

Comment:
isn't 50% inaccurate now? this should be 'less than ``replication_crit`` of a services...